config/locales/en.yml

```diff
@@ -611,17 +610,14 @@ en:
         - text: Find out payment dates and how to report a payment is missing.
           title: Cost of Living Payment
           href: /guidance/cost-of-living-payment
-          image_src: homepage/cost-of-living-november-2022.jpg
           image_src_new: homepage/cost-of-living.png
```

Comment:
Could these be renamed to image_src now since we don't have both image_source and image_source_new?

Reply:
Oh I see that's in the PR description, never mind

Reply:
I made a note in the follow up work section of the PR descriptions as there are quite a few that have a `_new` prefix.

Reply:
I can do it now if you think that's better?

Reply:
Actually I think it's fine splitting this up into multiple PRs because it'll be easier to review